modules/installation/pages/uyuni-proxy-setup.adoc

```diff
@@ -0,0 +1,252 @@
+
+
+
+[[uyuni-proxy-setup-pxe]]
+== Enable PXE Boot
+
+
+
+[[proxy.pxe.sync]]
+=== Synchronize Profiles and System Information
+
+To enable PXE boot through a proxy, additional software must be installed and configured on both the {productname} Proxy and the  {productname} Server.
+
+. On the {productname} Proxy, install [package]#susemanager-tftpsync-recv#:
++
+
+----
+zypper in susemanager-tftpsync-recv
+----
+
+. On the {productname} Proxy, run the [command]``configure-tftpsync.sh`` setup script and enter the requested information:
++
+
+----
+configure-tftpsync.sh
+----
++
+
+You will need to provide the hostname and IP address of the {productname} Server and of the proxy.
+You will also need to enter the path to the tftpboot directory on the proxy.
+
+. On the {productname} Server, install [package]``susemanager-tftpsync``:
++
+
+----
+zypper in susemanager-tftpsync
+----
+
+// same file name on the server and proxy!
+. On the {productname} Server, run [command]``configure-tftpsync.sh``.
+This creates the configuration, and uploads it to the {productname} Proxy:
++
+
+----
+configure-tftpsync.sh FQDN_of_Proxy
+----
+
+. Start an initial synchronization on the {productname} Server:
++
+
+----
+cobbler sync
+----
++
+
+The [command]``cobbler sync`` command is also useful after changing internal Cobbler settings.
+If you do not run Cobbler synchronization manually, it will run automatically when needed and thus may delay normal operation.
+For more information about Cobbler, see xref:client-configuration:cobbler.adoc[Cobbler].
+
+
+
+[[proxy.pxe.dhcp]]
+=== Configure DHCP for PXE through Proxy
+
+{productname} uses Cobbler for client provisioning.
+PXE (tftp) is installed and activated by default.
+Clients must be able to find the PXE boot on the {productname} Proxy using DHCP.
+Use this DHCP configuration for the zone which contains the clients to be provisioned:
+
+----
+next-server: <IP_Address_of_Proxy>
+filename: "pxelinux.0"
+----
+
+
+////
+// REMARK: 2019-08-23, ke: this needs closer checking
+[[replacing.a.susemgrproxy]]
+== Replace a {susemgrproxy}
+
+A proxy does not contain any information about the clients that are connected to it.
+Because of this, a proxy can be replaced by a new one at any time.
+The replacement proxy must have the same name and IP address as its predecessor.
+
+Leave the old proxy in place while you prepare the replacement.
+Create a reactivation key for this system and then register the new proxy using the reactivation key.
+If you do not use the reactivation key, you will need to re-register all the clients against the new proxy.
+
+[[proxy.migration3.replace]]
+.Procedure: Replacing a proxy and Keeping the Clients Registered
+. Before starting the actual migration procedure, save the data from the old proxy, if needed.
+Consider copying important data to a central place that can also be accessed by the new proxy.
+// REMARK: 2019-08-23, ke: this is unclear
```

Comment:
Same here, I think you need to add the URL to keep it in mind.

Reply:
it's part of the otther issue.